docs: update docs inline latest syntax changes (#4686)
confluentinc/ksql: docs/developer-guide/create-a-stream.md

### Create a Stream with Selected Columns

The following example creates a stream that has three columns from the

> `pageviews` topic: `viewtime`, `userid`, and `pageid`. All these columns will be loaded from the 

Reviewer: ```suggestion
`pageviews` topic: `viewtime`, `userid`, and `pageid`. All of these columns are loaded from the 
```

> {{ site.ak }} topic message value. If you wish to access data in your message key see 

Reviewer: ```suggestion
{{ site.ak }} topic message value. To access data in your message key, see 
```

the following CREATE STREAM statement into the CLI:

```sql
CREATE STREAM pageviews(
    viewtime BIGINT KEY,
    userid VARCHAR,
    pageid VARCHAR
  ) WITH (
    KAFKA_TOPIC='pageviews',
    VALUE_FORMAT='DELIMITED',
    PARTITIONS=4,
    REPLICAS=3
  );
```

> This will create the pageviews topics for you with the supplied partition and replica count. The 

Reviewer: ```suggestion
This statement creates the `pageviews` topic for you with the supplied partition and replica count. The 
```

> value of `viewTime` will be stored in the {{ site.ak }} topic message key. If you wish to store

Reviewer: ```suggestion
value of `viewTime` is stored in the {{ site.ak }} topic message key. To store
```

> the column in the message value, simple remove `KEY` from the column definition.

Reviewer: ```suggestion
the column in the message value, simply remove `KEY` from the column definition.
```
